atomic_reactor/plugins/exit_store_metadata.py

```diff
@@ -220,8 +219,8 @@ def run(self):
                     not self.workflow.dockerfile_images.base_from_scratch):
                 base_image_name = base_image
                 try:
-                    # OSBS2 TBD
-                    base_image_id = imageutil.base_image_inspect().get('Id', "")
+                    # OSBS2 TBD: inspect the correct architecture?
```

Comment:
well it is adding annotation annotations['base-image-id']

and as of now that ran also on the orchestrator so it was x86

but I am not sure if that information is used anywhere

Reply:
It looks like it's not used. Changed the TBD comment, we might want to remove it eventually